blueprints/ember-frost-test/index.js

```diff
@@ -1,13 +1,104 @@
+/**
+ * Install blueprint for ember-frost-test addon
+ */
+
+var chalk = require('chalk')
+var EOL = require('os').EOL
+
 module.exports = {
+
+  /**
+   * Add sinon-chai and chai-jquery bower libraries during testing
+   * @returns {Promise} a promise when the `ember-cli-build.js` file has been updated
+   */
+  addLoadingOfTestHelpers: function () {
+    const content =
+      '  if (app.env === \'test\') {' + EOL +
```

Comment:
I am thinking that we will want the `development` environment here as well for the cases when testing is done via the browser: `localhost:4200/tests`